doc/guides/gps_setup.md

```diff
@@ -0,0 +1,57 @@
+---
+module: [kind=guide] gps_setup
+---
+
+# Setting up GPS
+Some quick definitions for this page.
+```
+GPS host: a computer running the GPS program in host mode
+GPS constellation: a group of GPS hosts
+```
```

Comment:
I think this is probably better done as prose. So something like:

> The @{gps} API allows computers and turtles to find their current position using wireless modems.
> 
> In order to use GPS, you'll need to set up multiple _GPS hosts_. These are computers running the special `gps host` program, which tell other computers the host's position. Several hosts running together are known as a _GPS constellation._

Obviously feel free to tweak/rewrite this, just trying to give an idea :).